paper-dialog-behavior.html

```diff
@@ -205,32 +205,36 @@
     },
 
     _onFocus: function(event) {
-      if (this.modal) {
```

Comment:
Should we handle that case? There are many other things would need to be fixed in that case, see here 
http://jsbin.com/neyudo/2/edit?html,output
Can you open an issue for that?

I'll restore the if check in here.